partitura/score.py

```diff
@@ -28,7 +28,6 @@
 from partitura.utils import (
     ComparableMixin,
     ReplaceRefMixin,
-    iter_subclasses,
```

Comment:
Is `iter_subclasses` used somewhere else in the library? If not, I would suggest removing it completely

Reply:
It's not being used anywhere in the package. I also think we should remove it. As with the comment below, the function is not referenced in the official documentation.